Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -572,23 +589,20 @@ final class StatusTableViewController: LoopChartsTableViewController {
         }
     }
 
-    private enum Section: Int {
-        case hud = 0
+    private enum Section: Int, CaseIterable {
+        case tempStatus
```

Comment:
[nit] This is a very specific type of cell for "Alert Permissions Disabled", but given the generic and not-very-descriptive name of "tempStatus". Consider a more explicit name. If it ever becomes more generic and holds a variety of information we can change then.

Reply:
I picked `alertPermissionsDisabledWarning`.  Hope that's clearer, and not too verbose...